app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class RecipesController < ApplicationController
+    def show
+        @recipe = Recipe.find(params[:id])
+    end
+end
```

Comment:
This file seems to be using 4 spaces over 2, which is Ruby convention. Other than that, everything is great.